irohad/consensus/yac/impl/yac_hash_provider_impl.cpp

```diff
@@ -1,33 +1,22 @@
 /**
- * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
- * http://soramitsu.co.jp
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
 #include "consensus/yac/impl/yac_hash_provider_impl.hpp"
 #include "interfaces/iroha_internal/block.hpp"
+#include "interfaces/iroha_internal/proposal.hpp"
 
 namespace iroha {
   namespace consensus {
     namespace yac {
 
       YacHash YacHashProviderImpl::makeHash(
-          const shared_model::interface::Block &block) const {
+          const shared_model::interface::Block &block,
+          const shared_model::interface::Proposal &proposal) const {
         YacHash result;
-        auto hex_hash = block.hash().hex();
-        result.proposal_hash = hex_hash;
-        result.block_hash = hex_hash;
+        result.proposal_hash = proposal.hash().hex();
+        result.block_hash = block.hash().hex();
```

Comment:
No need to change this method in this pull request if other parts of Yac are not changed.